src/OpenTracing/NoopTracer.php

```diff
@@ -12,24 +13,21 @@ public static function create()
         return new self();
     }
 
-    /** @return Span */
     public function startSpan(
         $operationName,
         SpanReference $parentReference = null,
         $startTimestamp = null,
         Tag ...$tags
     ) {
-        // TODO: Implement startSpan() method.
+        return NoopSpan::create();
```

Comment:
Is there a reason you prefer `NoopSpan::create()` over `new NoopSpan()`? To me it just seems like an unneeded function call. Static methods are inherited in PHP, so they fall under Liskov's substitution principle, while constructors do not. So you couldn't have a child class taking different constructor parameters without breaking the substitution principle in`create()`.